### ISSUE TITLE
REID Center Loss Learning Rate

### ISSUE THREAD

**Issue by awarebayes** [PaddlePaddle/PaddleClas#2887]
In file `ppcls/configs/reid/strong_baseline/softmax_triplet_with_center.yaml`
 ```yml
Optimizer:
  - Adam:
      scope: RecModel
      lr:
        name: Piecewise
        decay_epochs: [30, 60]
        values: [0.00035, 0.000035, 0.0000035]
        warmup_epoch: 10
        warmup_start_lr: 0.0000035
        by_epoch: True
        last_epoch: 0
      regularizer:
        name: "L2"
        coeff: 0.0005
  - SGD:
      scope: CenterLoss
      lr:
        name: Constant
        learning_rate: 1000 # NOTE: set to ori_lr*(1/centerloss_weight) to avoid manually scaling centers' gradidents.
```
Number 1000 seems to big to be used as a learning rate for sgd. I think it needs to be changed.
It is not clear what ori_lr refers to... Is it 0.0000035 in adam?

**Comment by HydrogenSulfate** [comment]
It is because that softmax_triplet_with_center use two optimizer for **parametrized** center loss(with learnable `centers`) and model. So in original author michuanhaohao's repo, he manually scaling gradient of center after backward([code here](https://github.com/michuanhaohao/reid-strong-baseline/blob/master/engine/trainer.py#L89-L90)), therefore the actual learning rate(ori_lr) for centers is `global_lr * (1/center_weight)`, that is to say, learning rate is kept unchagned for model but scaling up `(1/center_weight)` times only for centers.

In PaddleClas, we can not add this tricky code because it is not a common usage, so we consider of an equivalent implementation. We move scaling factor `(1/center_weight)` from center loss weight to learning rete of it's optimizer(SGD), so the SGD's learning rate is set to `_C.SOLVER.CENTER_LR] * (1/center_weight) = 0.5 * (1/0.0005) = 1000.0`.

[_C.SOLVER.CENTER_LR](https://github.com/michuanhaohao/reid-strong-baseline/blob/3da7e6f03164a92e696cb6da059b1cd771b0346d/config/defaults.py#L105)

**Comment by awarebayes** [comment]
Thanks a lot
